bin/tass

```diff
@@ -57,10 +57,17 @@ when 'create'
   end
 
   config = initialize_environment(opts)
+  sfr_config = {}  # @TODO: remove this when SFR config becomes sane
 
   # check the verison of the yaml we are reading, and do the new hotness if we are running v2 or higher
   if newer_yaml_version?(config)
     # Determine if we are making a cloudformation request
+    if config[:stack][:spot_fleet]
```

Comment:
Just thinking ahead under the assumption were going to want support things like asgs and launch configs outside the scope of being in a cf stack. Not something to worry about for the time being but we'll have to address it in the future.